astropy/modeling/tests/test_fitters.py

```diff
@@ -501,6 +501,21 @@ def test_param_cov(self):
         assert_allclose(fmod.parameters, beta.A.ravel())
         assert_allclose(olscov, fitter.fit_info['param_cov'])
 
+    def test_minimize(self):
+        """
+        Runs MinimizeFitter with most methods.
+        """
+        # get rid of tiny bound so the method which don't support
+        # bounds can be tested
+        self.gauss.bounds['stddev'] = (None, None)
+        for meth in ['Nelder-Mead', 'Powell', 'CG', 'BFGS', 'L-BFGS-B', 'TNC', 'SLSQP']:
+            mm = MinimizeFitter(meth)
+            res=mm(self.gauss, self.xdata, self.ydata)
```

Comment:
I think you want to change the initial model a little. Right now, this test doesn't do much since it compares whether the result of the optimizer is close to the initial guess.

Reply:
Maybe you also want to assert that `res['status']` is meaningful?